lib/matplotlib/contour.py

```diff
@@ -1749,7 +1752,7 @@ def _initialize_x_y(self, z):
             however introduce rendering artifacts at chunk boundaries depending
             on the backend, the *antialiased* flag and value of *alpha*.
 
-        linewidths : float or sequence of float, default: :rc:`lines.linewidth`
+        linewidths : float or sequence of float
```

Comment:
I'd leave the default:
```suggestion
        linewidths : float or sequence of float, default: rc:`contour.linewidth`
```